examples/eta/README.md

```diff
@@ -0,0 +1,21 @@
+# eta
+
+Example of how to use Opine's template engine and rendering capabilities with the `eta` template engine.
+
+View Eta's documentation at https://eta.js.org
+
+## How to run this example
+
+Run this example using:
+
+```bash
+deno run --allow-net --allow-read ./examples/eta/index.ts
+```
+
+if have the repo cloned locally _OR_
+
+```bash
+deno run --allow-net --allow-read https://raw.githubusercontent.com/asos-craigmorten/opine/main/examples/eta/index.ts
+```
+
+if you don't!
```

Comment:
Have found that static file serving doesn't work currently with remote running (likely because it looks for the folder in the local fs - need to check).

```suggestion
after cloning the repo locally.
```

Reply:
Just updated the README :)